src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles the loading and saving of data in file,
+ * where data represents the list of tasks
+ * to be done
+ */
+
+
```

Comment:
I think the coding style guide says there should not be white spaces between the JavaDoc and the class/method.